src/app/pages/details/desktop-view/promo.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import RawHTML from '~/components/jsx-helpers/raw-html';
+
+type PromoteData = {
+    content?: {
+        description: string;
+    }
+}
+
+export default function Promo({promoteSnippet}: {promoteSnippet: PromoteData | PromoteData[]}) {
+    // promoteSnippet may or may not be in an array
+    const obj = promoteSnippet;
```

Comment:
just curious why you need to assign this to a variable?

Reply:
It's a carryover from when it was written a bit differently. Updated.